client/src/components/LeafSpinner/LeafSpinner.js

```diff
@@ -3,8 +3,8 @@ import React from "react";
 
 import { primaryColor, secondaryColor } from "src/core/color_defs";
 
-import leaf_loading_spinner from "src/svg/leaf-loading-spinner.svg";
-import "src/components/SpinnerWrapper/LeafSpinner.scss";
+import leaf_loading_spinner from "./LeafSpinner.svg";
```

Comment:
Are we putting SVGs across the site now?

Reply:
The svgs in `src/svg` are specifically copied over by build base so that they can be referenced via static URL (for `img` tags, in CSS, etc). This svg never was, it was always imported (and therefore inlined in to webpack bundles), never needed to be in `src/svg`.

The biggest reason for moving it was that the svg has a number of classes from `LeafSpinner.scss` hardcoded in it. So,
  a) it can't be used properly unless those styles are also loaded
  b) they need to be kept in sync
Given that, _not_ collocating them is unsafe.

We have other svgs outside of `src/svg`, just already inside JS or stored as yaml. This just happens to be an svg instead :shrug: